CONTRIBUTING.rst

```diff
@@ -0,0 +1,12 @@
+.. _contributing:
+
+============
+Contributing
+============
+
+Before contributing code, please note:
```

Comment:
That could be neat to add a short sentence saying "Thanks for considering submitting a pull request, we really appreciate that. Before doing so, here are a few guidelines"